tests/test_core.py

```diff
@@ -374,9 +374,14 @@ def get_bullet_library(self, use_cmake):
   @also_with_standalone_wasm
   def test_hello_world(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_hello_world')
+    # must not emit this unneeded internal thing
+    self.assertNotContained('EMSCRIPTEN_GENERATED_FUNCTIONS', open('src.c.o.js').read())
 
-    src = open('src.c.o.js').read()
-    assert 'EMSCRIPTEN_GENERATED_FUNCTIONS' not in src, 'must not emit this unneeded internal thing'
+  @no_asmjs()
+  def test_wasm_syncronous_compilation(self):
+    self.set_setting('WASM_ASYNC_COMPILATION', 0)
```

Comment:
can use `@sync` instead of this line (but not sure if that's better or not, either way)

Reply:
Done